Plugin~/WebRTCPlugin/Codec/NvCodec/NvEncoderImpl.cpp

```diff
@@ -408,22 +468,30 @@ namespace webrtc
             m_configurations[0].SetStreamState(false);
             return;
         }
-        float fps = static_cast<float>(parameters.framerate_fps + 0.5);
-        int32_t pixelCount = m_codec.width * m_codec.height;
 
-        auto requireLevel = H264SupportedLevel(pixelCount, fps);
+        m_codec.maxFramerate = static_cast<uint32_t>(parameters.framerate_fps);
 
-        if (!requireLevel || m_level < static_cast<NV_ENC_LEVEL>(requireLevel.value()))
+        // todo(kazuki): Not supported framerate adjusting
+        m_codec.maxFramerate = 30;
```

Comment:
I think need to use `parameters` to determine the Bitrate and overwrite m_codec.maxBitrate.

```suggestion
        m_codec.maxFramerate = 30;
        m_bitrateAdjuster->SetTargetBitrateBps(parameters.bitrate.get_sum_bps());
        m_codec.maxBitrate = m_bitrateAdjuster->GetTargetBitrate();
```